sql/ttl/ttljob: TestRowLevelTTLJobMultipleNodes failed

sql/ttl/ttljob.TestRowLevelTTLJobMultipleNodes [failed](https://mesolite.cluster.engflow.com/invocations/default/272f2387-ce99-4149-bab0-035aefa37ebc?testReportRun=2&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL3R0bC90dGxqb2I6dHRsam9iX3Rlc3Q=) on release-25.2 @ [5dda939d2b9e10439a62bea35ce07af749c08906](https://github.com/cockroachdb/cockroach/commits/5dda939d2b9e10439a62bea35ce07af749c08906):


```
=== RUN   TestRowLevelTTLJobMultipleNodes
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestRowLevelTTLJobMultipleNodes3167273027
    test_log_scope.go:76: use -show-logs to present logs inline
    ttljob_test.go:593: -- test log scope end --
test logs left over in: outputs.zip/logTestRowLevelTTLJobMultipleNodes3167273027
--- FAIL: TestRowLevelTTLJobMultipleNodes (307.27s)
=== RUN   TestRowLevelTTLJobMultipleNodes/2_splits
    test_server_shim.go:90: cluster virtualization disabled due to issue: #109391 (expected label: C-bug)
    ttljob_test.go:153: condition failed to evaluate within 3m0s: from ttljob_test.go:168: 
        expectedStatus="succeeded"
        actualStatuses="failed"
         expectedError=""
          actualErrors="incorrect number of span partitions expected=3 actual=1"
--- FAIL: TestRowLevelTTLJobMultipleNodes/2_splits (237.03s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRowLevelTTLJobMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55780

--- a/pkg/sql/ttl/ttljob/ttljob_test.go
+++ b/pkg/sql/ttl/ttljob/ttljob_test.go
@@ -594,6 +594,8 @@ func TestRowLevelTTLJobMultipleNodes(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderRace(t, "the test times out under race")
+
 	testCases := []struct {
 		desc     string
 		splitAts []int

